src/components/Regions.js

```diff
@@ -247,6 +265,9 @@ const RegionTable = (props) => {
           </>
         )}
         <TableCell align="right" className={classes.operations}>
+          <IconButton title="download" onClick={() => handleDownload(item)}>
+            <CloudDownload />
```

Comment:
For consistency with the _Earnings_ tool, let's use the `Download` icon. (_Earnings_ actually uses `GetApp`, but that's just an alias for `Download`).

Reply:
Thank you @nmcharlton. There is a new commit where the icon has been changed. I appreciated your clarification about the alias, I simply couldn't find the icon in the node modules but, I did learn something new.